drivechain-server/config.go

```diff
@@ -19,6 +19,8 @@ type Config struct {
 
 	ElectrumHost  string `long:"electrum.host" description:"host:port for connecting to Electrum server" default:"electrum.blockstream.info:60002"`
 	ElectrumNoSSL bool   `long:"electrum.no-ssl" description:"Avoid using SSL for connecting to Electrum"`
+
+	Passphrase string `long:"passphrase" description:"Passphrase for encrypting/decrypting the wallet key"`
```

Comment:
Can we add a default here? It's only going to be used with test-money for the overseeable future.

Reply:
nvm